snippets/emulated-nrf54l05/README.rst

```diff
@@ -0,0 +1,20 @@
+.. _emulated-nrf54l05:
+
+nRF54L05 snippet
+################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+Simulate the nRF54L05 on an nRF54L15.
+
+Supported SoCs and boards
+*************************
+
+Currently, the only SoC and board supported for use with the snippet is:
```

Comment:
It should be pointed out that this snippet is not compatible with FLPR snippets and target.
Same with L10.

Reply:
Yep, right now it's only in the commit message but you are right.